.eslintignore

```diff
@@ -2,5 +2,7 @@ dist/
 node_modules/
 e2e/__cases__/
 e2e/__workdir_synlink__/
+e2e/__external-repos__/
 coverage/
 docs/
+*.config.js
```

Comment:
this needs a new line at the end

Reply:
Added a line break in the latest commit now.

<img width="423" alt="Skjermbilde 2020-05-03 kl  13 56 02" src="https://user-images.githubusercontent.com/42898343/80913570-d55b1f00-8d45-11ea-89b5-3fa127fbb29c.png">